src/organization/screens/repository-list.screen.js

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable no-shadow */
```

Comment:
refactor: This screen should be extracted to `common/screens/` and be used in **all** repositories list screen.